tests/io/write_mesh_particles.cc

```diff
@@ -15,7 +15,7 @@ bool write_json(unsigned dim, bool resume, const std::string& analysis,
   auto io_type = "Ascii2D";
   std::string material = "LinearElastic2D";
   std::vector<double> gravity{{0., -9.81}};
-  unsigned material_id = 1;
+  unsigned material_id = 0;
```

Comment:
So the plan is to initialize with `material_id` = 0, then update it with the `material_set` to make it 1?

Reply:
Correct. When `material_id: 1` then `all tests pass`.